pygyw/layout/helpers.py

```diff
@@ -195,7 +197,7 @@ def justify(text: str, width: int):
     return lines
 
 
-def rgba8888_bytes_from_color_string(color: str) -> bytes:
+def rgba8888_bytes_from_color_string(color: str | None) -> bytes:
     """Transform an ARGB color string into an RGBA8888 byte array of length 4."""
 
     if color is None:
```

Comment:
If the color is None, shouldn't we use "our NULL symbol" `NULLNULL` instead ?

Reply:
For the "old" commands it's the case, but for rectangle and spinner drawings, it's sent as a 4 byte rgba color because it's more machine friendly, it takes less space, and I don't need to do any conversion in the firmware. In this case, alpha == 0 is the new "null" color because it's useless to have a fully transparent color, so I just send (0, 0, 0, 0).